cellfinder/core/tools/source_files.py

```diff
@@ -1,9 +1,27 @@
 from pathlib import Path
 
 
-def source_config_cellfinder():
+def default_configuration_path():
+    """
+    Returns the default configuration path for cellfinder.
+
+    Returns:
+        Path: The default configuration path.
+    """
     return Path(__file__).parent.parent / "config" / "cellfinder.conf"
 
 
-def source_custom_config_cellfinder():
-    return Path(__file__).parent.parent / "config" / "cellfinder.conf.custom"
+def user_specific_configuration_path():
+    """
+    Returns the path to the user-specific configuration file for cellfinder.
+
+    This function returns the path to the user-specific configuration file
+    for cellfinder. The user-specific configuration file is located in the
+    user's home directory under the ".cellfinder" folder and is named
+    "cellfinder.conf.custom".
+
+    Returns:
+        Path: The path to the custom configuration file.
+
+    """
+    return Path.home() / ".cellfinder" / "cellfinder.conf.custom"
```

Comment:
I seem to recall we have an outstanding issue to standardise where we put user configs; and want to say we ended up settling on the `~/.brainglobe` directory. So maybe this should be `~/.brainglobe/cellfinder`? Though I don't know what the current default directory for `brainmapper` is set to.

Reply:
Good point... (relevant issue is https://github.com/brainglobe/BrainGlobe/issues/26) - I suggest tackling that in one go, and for now sticking to the "old" convention?